br/br-batch-create-table.md

```diff
@@ -0,0 +1,65 @@
+---
+title: BR Batch Create Table
+summary: Learn how to use BR batch create table feature. When restoring data, BR can use batch create table feature to speed up the restoration.
+---
+
+# BR Batch Create Table
+
+When restoring data using Backup & Restore (BR), BR creates the databases and tables in the target TiDB first, then starts restoring data. In the versions earlier than TiDB v6.0.0, BR uses the [serial execution](#implementation-principles) scheme to create tables in the restoration process. However, when restoring data with a large number (nearly 50000) of tables, this scheme takes much time to create tables.
+
+To speed up the table creation process, and thereby reduce the time for restoring data, the BR batch create table feature is introduced in TiDB v6.0.0. This feature is enabled by default.
+
+> **Note:**
+>
+> - To use the BR batch create table feature, both TiDB and BR should be in 6.0.0 or later versions. If either TiDB or BR is in the version lower than 6.0.0, BR uses the serial execution scheme.
+> - Suppose that you use a cluster management tool (for example, TiUP), and your TiDB and BR are in 6.0.0 or later versions. In this case, BR enables batch create table feature by default without additional configuration.
+
+## User scenario
+
+When you need to restore data with a considerable number of tables, for example, 50000 tables, you can use BR batch create table feature to speed up the restoration process.
+
+For the detailed effect, see [Test batch create table feature](#test-batch-create-table).
+
+## Use batch create table
+
+BR enables batch create table feature and configures `--ddl-batch-size=128` by default in 6.0.0 or later versions. Therefore, you do not need to configure this parameter additionally. `--ddl-batch-size=128` means that BR creates tables in multiple batches, and each batch has 128 tables.
+
+To disable this feature, you can set `--ddl-batch-size` to `0` by the following command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full -s local:///br_data/ --pd 172.16.5.198:2379 --log-file restore.log --ddl-batch-size=0
+```
+
+After disabling the feature, BR uses the [serial execution scheme](#implementation-principles) instead.
+
+## Implementation principles
+
+- Serial execution scheme before v6.0.0:
+
+    In the versions earlier than 6.0.0, BR uses the serial execution scheme. When restoring data, BR creates the databases and tables in the target TiDB first, then starts restoring data. To create tables, after calling TiDB API, BR uses the SQL statement `Create Table`. TiDB DDL owner creates tables sequentially. Once the DDL owner creates a table, the schema version changes correspondingly, and each version change synchronizes to other BRs and other TiDB DDL workers. Hence, when restoring a large number of tables, the serial execution scheme takes too much time.
```

Comment:
```suggestion
    In the versions earlier, BR uses the serial execution scheme. When restoring data, BR creates the databases and tables in the target TiDB first, then starts restoring data. To create tables, BR calling TiDB internal API, more like BR uses the SQL statement `Create Table`. TiDB DDL owner creates tables sequentially. Once the DDL owner creates a table, the DDL schema version changes correspondingly, and each version change synchronizes to other TiDB DDL workers (include BR). Hence, when restoring a large number of tables, the serial execution scheme takes too much time.
```

Reply:
The corresponding Chinese part is going to be updated.

Reply:
> 

The following part in Chinese is going to be updated.